frontend/src/shared/components/tasque-task-editing/task-editing.component.ts

```diff
@@ -334,4 +346,42 @@ export class TaskEditingComponent extends BaseComponent implements OnInit {
         this.clearForm();
       });
   }
+
+  addUser(email: string): void {
+    const isUserAdded = this.board.users.find((user) => user.email == email) ? true : false;
+
+    if (isUserAdded) {
+      this.notificationService.error('User with given email has already been added');
+      return;
+    }
+
+    this.userService.getUserByEmail(email)
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe(
+        (user) => {
+          if (!user.body) {
+            return;
+          }
+          this.editTaskForm.controls.assignees.value.push(user.body);
+          this.board.users.push(this.convertToUserCard(user.body));
+        }
+      );
+  }
+
+  deleteUser(email: string): void {
+    const boardIndex = this.board.users.findIndex((x) => x.email == email);
+    this.board.users.splice(boardIndex, 1);
+    const index = this.editTaskForm.controls.assignees.value.findIndex((x: UserModel) => { x.email == email; });
+    this.editTaskForm.controls.assignees.value.splice(index, 1);
+  }
+
+  private convertToUserCard(user: UserModel): IUserCard {
```

Comment:
Could you add a TODO comment here ? When I start working on a redisign of select-users, I'll expand UserModel and get rid of IUserCard

Reply:
Added TODO.